src/net/sourceforge/kolmafia/request/MrStore2002Request.java

```diff
@@ -21,8 +22,24 @@ public class MrStore2002Request extends CoinMasterRequest {
           .withProperty("availableMrStore2002Credits")
           .withShopRowFields(master, "mrstore2002");
 
+  // True if accessing the shop via using the item and redirecting. The
+  // first time you do this per day, KoL adds your daily store credits.
+  // False if going directly to the shop.
+  private boolean using = false;
+
   public MrStore2002Request() {
     super(MR_STORE_2002);
+    // If we have not yet obtained today's store credits, "visit" by
+    // using the item (which redirects to the shop), rather than going
+    // to the shop directly, since the latter will not collect credits.
+    if (!Preferences.getBoolean("_2002MrStoreCreditsCollected")) {
+      int itemId =
+          KoLCharacter.inLegacyOfLoathing()
+              ? ItemPool.REPLICA_MR_STORE_2002_CATALOG
+              : ItemPool.MR_STORE_2002_CATALOG;
```

Comment:
It's possible to be in LoL and have the main catalog and not the replica:
* don't buy the replica
* break ronin
* pull the main catalog

Fringe, but...

Reply:
Can you have both? If so, does it matter which you use?
Once you free the king in LoL, all replica items are removed, correct?
```
  public static String accessible() {
    if (InventoryManager.hasItem(ItemPool.MR_STORE_2002_CATALOG)) {
      return null;
    }
    if (KoLCharacter.inLegacyOfLoathing()
        && InventoryManager.hasItem(ItemPool.REPLICA_MR_STORE_2002_CATALOG)) {
      return null;
    }
    return "You need a 2002 Mr. Store Catalog in order to shop here.";
  }
```
Seems like simple possession of either catalog gives you access and we should use whichever one you actually have.

Note that "visit" doesn't actually check "accessible".

Reply:
Correct. You can have both, I expect, but if you have the replica you're definitely in Legacy of Loathing.